packages/sit-onyx/playwright.config.ts

```diff
@@ -41,7 +41,7 @@ export default defineConfig({
   },
   /* Configure projects for major browsers */
   projects: [
-    { name: "chromium", use: { ...devices["Desktop Chrome"] } },
+    { name: "Microsoft Edge", use: { ...devices["Desktop Edge"], channel: "msedge" } },
```

Comment:
```suggestion
    { name: "edge", use: { ...devices["Desktop Edge"], channel: "msedge" } },
```

For the other browsers is also just the browser name and lowercase :) This name will be added to the screenshot names so I would keep it short here